RiotSharpTest/StaticRiotApiTest.cs

```diff
@@ -254,7 +286,7 @@ public void GetSummonerSpellsAsync_Test()
         public void GetSummonerSpell_Test()
         {
             var spell = api.GetSummonerSpell(StaticRiotApiTestBase.region, 
-                StaticRiotApiTestBase.staticSummonerSpell, SummonerSpellData.all);
+                (int) StaticRiotApiTestBase.staticSummonerSpell, SummonerSpellData.all);
```

Comment:
My reason behind this change is, that currently you get the summoner spell id as an int back (in the match endpoint). Therefore, it is more fluent to adjust the static api to also use an int.